src/modules/comment.js

```diff
@@ -8,6 +8,27 @@ const closePopUp = () => {
   comment.innerHTML = '';
 };
 
+const renderComments = async (showId, container) => {
+  container = container.querySelector('ul');
+  container.innerHTML = '';
+  try {
+    const reservations = await fetchcomments(showId);
+    reservations.forEach((comments) => {
+      const item = document.createElement('li');
+      item.innerHTML = `
+        ${comments.date_start} to ${comments.date_end} by ${comments.username}
```

Comment:
Dear @saifullah767 please double-check that the Involvement API returns the comments with those properties and kindly update it accordingly.